components/form/UploadImageInput.tsx

```diff
@@ -10,117 +10,90 @@ type UploadImageProps = {
   setError?: (error: string) => void;
   isRequired?: boolean;
   imageType: 'item' | 'profile';
+  name: string;
 };
 
 const CDN = `${process.env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/images/`;
 
```

Comment:
The scope of this refactor is really outside the scope of this issue and removes some functionality that we don't want to lose, like removing a user profile picture they already have one. This component is already scheduled for a refactor, see issue #334. Please revert the changes on this file.